src/lib.rs

```diff
@@ -8,24 +8,23 @@
 //! ```
 //! use filenamify::filenamify;
 //! let safe_filename = filenamify("//foo/bar/file");
-//! println!("{}", safe_filename); // Prints "_foo_bar_file"
+//! assert_eq!(safe_filename, "_foo_bar_file");
 //! ```
-//!
+
+use std::sync::OnceLock;
```

Comment:
This will bump the MSRV to `1.70.0`, while the crate's current MSRV is `1.40.0`

Reply:
You are right, I forgot about MSRV compatibility.

I will rewrite it using `lazy_static` soon, that's gonna keep MSRV unchanged